src/PhpParser/NodeTraverser/RectorNodeTraverser.php

```diff
@@ -46,6 +53,31 @@ public function refreshPhpRectors(array $rectors): void
         $this->areNodeVisitorsPrepared = false;
     }
 
+    /**
+     * We return the list of visitors (rector rules) that can be applied to each node class
+     * This list is cached so that we don't need to continually check if a rule can be applied to a node
+     *
+     * @return NodeVisitor[]
+     */
+    protected function getVisitorsForNode(Node $node): array
```

Comment:
:+1: 

Could you add few test for this method? Just so we have it under control.

Reply:
This is a protected method. What do you use to test protected methods?